bitnami/thanos/values.yaml

```diff
@@ -2582,6 +2582,566 @@ metrics:
     ##
     # scrapeTimeout: 10s
 
+  ## PrometheusRule CRD configuration
+  ##
+  prometheusRule:
+    enabled: false
+    ## Namespace in which the PrometheusRule CRD is created
+    ##
+    namespace: monitoring
+    ## List of rules for each component
+    ##
+    rules:
+      ##
+      ## Compactor Prometheus Rules
+      ##
+      compactor:
```

Comment:
I think that maybe it is a good idea to use strings, something like:
```
rules:
  compactor: | 
```
and use the common.tplvalues.render. Same with query, receive, etc

Reply:
Hi @alvneiayu 
This was updated, thank you for your review. 
Happy to contribute to bitnami charts, we've been using them a lot.

François